salt/modules/lxd.py

```diff
@@ -1427,7 +1417,14 @@ def container_device_add(
     """
     container = container_get(name, remote_addr, cert, key, verify_cert, _raw=True)
 
-    kwargs["type"] = device_type
+    if "type" not in kwargs:
+        kwargs["type"] = "disk"
+
+    # retro compatibility
+    if "device_type" in kwargs:
```

Comment:
@Ch3LL this will keep the device_type working (or rather: makes it actually work)

Reply:
We will need to add a deprecation message if `device_type` is used as documented here: https://docs.saltproject.io/en/latest/topics/development/deprecations.html

Then we can remove this kwarg in a couple releases.